src/utils.rs

```diff
@@ -64,3 +71,17 @@ pub fn get_human_readable_timestamp(duration: Duration) -> String {
 pub fn get_full_username(user: &User) -> String {
     format!("{}#{:04}", user.name, user.discriminator)
 }
+
+pub fn get_prefixes() -> serde_json::Value {
+    let data = fs::read_to_string("prefixes.json").unwrap_or_else(|error| {
+        if error.kind() == ErrorKind::NotFound {
+            let mut f = File::create("prefixes.json").expect(FAILED_CREATE_PREFIXES);
+            f.write_all("{}".as_bytes()).expect(FAILED_SAVE_PREFIXES);
+            "{}".to_string()
+        } else {
+            panic!("Problem opening the file: {:?}", error);
+        }
+    });
```

Comment:
Take a look into https://doc.rust-lang.org/std/fs/struct.OpenOptions.html#examples

Reply:
Sure, but since in this case I need the file initialised with something, it is more useful with the current implementation.

Reply:
I have no time now, but please let's hold this until tonight as I might have an idea. If that does not work, this is not by all means bad or wrong, so in that case we consider this resolved.

Reply:
What are we doing tonight? 😏